packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -364,6 +364,31 @@ Sample user data:
 | Workday ID                                                         | `hasWorkdayUser() ? findWorkdayUser().employeeID : null`                                                                                                 | 123456                  | Check if user has a Workday assignment, and if so, return their Workday employee ID.                                                                                                                                                                                                                                                                                                                   |
 | Active Directory UPN                                               | `hasDirectoryUser() ? findDirectoryUser().managerUpn : null`                                                                                             | bob@okta.com            | Check if user has an Active Directory assignment, and if so, return their Active Directory manager UPN.                                                                                                                                                                                                                                                                                                |
 
+
+## Expressions for custom claims
+
+Okta provides a few expressions that you can only use with custom claims.
+* See [Create claims](/docs/guides/customize-authz-server/create-claims).
+* See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).
+
+| Syntax           | Definitions                                                             | Examples     |
+| --------         | ----------                                                              | ------------ |
+| `app.$attribute` | `app` refers to the name of the OIDC app.<br>`$attribute` refers to the attribute variable name. | app.id<br>app.clientId<br>app.profile |
+| `access.scope` | `access` refers to the access token requesting the Scopes.<br>`scope` refers to the array of granted Scopes. | access.scope |
+
+### Samples
+
+#### Sample using app attributes
+To include an app Profile label, use the following expression:<br>
+`app.profile.label`
+
+See [Include app-specific information in a custom claim](/docs/guides/customize-tokens-returned-from-okta/create-app-profile-attribute).
+
+#### Sample using access.scope
+To include a granted Scope array and convert it to a space-delimited string, use the following expression:<br>
```

Comment:
You can name a claim `scope` in API Access Management custom authorization servers. You can then use the EL expression `access.scope` in custom claims to return an array of granted scope strings.

To include a granted Scope array and convert it to a space-delimited string, use the following expression:<br>
`String.replace(Arrays.toCsvString(access.scope),","," ")`

Reply:
do we use scope with a capital S in other places in the documentation? I feel we should use scope instead of Scope unless you made it this way for a particular reason.

Reply:
As we discussed, we might need to chat with Susan about capitalization of "scope", but I've remove the capital 'S' for now :)